Removed mentioning of "Identity Document" (#259)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

~~~aasvg
 .----------.
|  Artifact  |

>  '-----+----'            .--------------.
>        v                |  Credentials  |
>   .----+----.           | Trust Anchors |
>  | Statement |           '----+--+------'

Reviewer: ```suggestion
 '-----+----'            .-------------.
       v                |  Credentials  |
  .----+----.           | Trust Anchors |
 | Statement |           '----+--+-----'
```

Multiple Issuers can make the same Statement about a single Artifact, affirming multiple Issuers agree.

> At least one identifier to a credential MUST be included in the protected header of the COSE Envelope, as one of `x5t` or `kid`.

Reviewer 2: ```suggestion
At least one identifier representing one credential MUST be included in the protected header of the COSE Envelope, as one of `x5t` or `kid`.
```
